integrations/salesforce/checks.go

```diff
@@ -0,0 +1,43 @@
+package salesforce
+
+import (
+	"context"
+
+	"go.autokitteh.dev/autokitteh/integrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+// Status checks the connection's initialization status (is it
```

Comment:
The first word in Go function docs needs to be the function name, and it needs to match even if it's lower-case.
```suggestion
// status checks the connection's initialization status (is it
```